src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,29 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, hidden = false) {
+    this.name = name;
+    this.health = 100;
+    this.hidden = hidden;
+    Animal.alive.push(this);
```

Comment:
```suggestion
  constructor(name) {
    this.name = name;
    this.health = 100;
    Animal.alive.push(this);
```

only Herbivores should have `hidden` property
and don't pass it as a parameter, just ad to the constructor with default value